ush/python/pygfs/task/archive.py

```diff
@@ -312,6 +323,22 @@ def _gen_relative_paths(self, root_path: str) -> Dict:
 
         return rel_path_dict
 
+    @staticmethod
+    @logit(logger)
+    def _construct_arcdir_set(j2yaml, arch_dict) -> Dict:
```

Comment:
This should have a doc-block

Reply:
Yes it should.  I've added a block and inline comments.